ts/navigation/AuthenticatedStackNavigator.tsx

```diff
@@ -285,13 +286,9 @@ const AuthenticatedStackNavigator = () => {
           ...hideHeaderOptions
         }}
       />
-      {/* 
-        This screen is outside the WalletPaymentNavigator to enable the slide from bottom animation.
-        FIXME IOBP-383: Using react-navigation 6.x we can achive this using a Stack.Group inside the WalletPaymentNavigator
-      */}
       <Stack.Screen
-        name={WalletPaymentRoutes.WALLET_PAYMENT_BARCODE_SCAN}
-        component={WalletPaymentBarcodeScanScreen}
+        name={WalletBarcodeRoutes.WALLET_BARCODE_MAIN}
```

Comment:
why remove the comment? unless the issue is not relevant anymore, ofc

Reply:
The issue is not relevant anymore :)